crates/forge-runner/src/lib.rs

```diff
@@ -0,0 +1,437 @@
+use crate::fuzzer::RandomFuzzer;
```

Comment:
This file is messy, it has +400 lines and is related to many things. IMO we should split it

Reply:
#1091 

It will be done but in a separate PR